lib/dialects/abstract/query-generator.js

```diff
@@ -1754,10 +1754,14 @@ var QueryGenerator = {
    */
   addLimitAndOffset: function(options, model) {
     var fragment = '';
-    if (options.offset && !options.limit) {
+    /*jshint eqeqeq:false*/
+    var limitExist = options.limit != null;
+    var offsetExit = options.offset != null;
```

Comment:
Sorry for the bikeshedding, but i actually find the other style more readable :)